superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -220,6 +224,15 @@ export default class ResultSet extends React.PureComponent<
       const { showSaveDatasetModal } = this.state;
       const { query } = this.props;
 
+      const dataset: ISaveableDataset = {
```

Comment:
@eric-briscoe what does the convention of naming with a prefix of `I` mean? interface?